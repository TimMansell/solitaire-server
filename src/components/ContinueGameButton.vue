<template>
  <Button @click="continueGame" data-test="continue-game-btn">
    Continue
  </Button>
</template>

<script>
import { mapActions } from 'vuex';
import Button from './Button.vue';

export default {
  name: 'NewGame',
  components: {
    Button,
  },
  methods: {
    ...mapActions(['toggleNewGame']),
    continueGame() {
      this.toggleNewGame();
    },
  },
};
</script>
