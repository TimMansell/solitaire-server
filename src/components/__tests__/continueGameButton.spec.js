import { shallowMount } from '@vue/test-utils';
import ContinueGameButton from '@/components/ContinueGameButton.vue';

describe('ContinueGameButton.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(ContinueGameButton);

    expect(wrapper).toMatchSnapshot();
  });
});
