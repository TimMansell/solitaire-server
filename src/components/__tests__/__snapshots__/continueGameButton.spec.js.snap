// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContinueGameButton.vue matches snapshot 1`] = `
<button-stub type="default" size="md" data-test="continue-game-btn">
  Continue
</button-stub>
`;
